### ISSUE TITLE
Could I get the project file of the model? Like a blender project.

### ISSUE THREAD

**Issue by chudonghao** [lettier/3d-game-shaders-for-beginners#15]
I want to port the project to other platforms instead of Panda3D.

**Comment by eoinroe** [comment]
Yes this would be great if possible!  Using Panda 3D on macOS running Mojave or later can be a tricky.  I would love to recreate the project using Metal instead.  Unfortunately there is not much information online regarding converting bam files to other formats...

**Comment by jfromell** [comment]
+1 on this, even just the OBJs would be great.

**Comment by gonnavis** [comment]
On win10, I successed with:
```
C:\Panda3D-1.10.7-x64\bin>bam2egg C:\***\3d-game-shaders-for-beginners_github\demonstration\eggs\mill-scene\mill-scene.bam C:\***\3d-game-shaders-for-beginners_github\demonstration\eggs\mill-scene\mill-scene.egg
C:\Panda3D-1.10.7-x64\bin>egg2obj C:\***\3d-game-shaders-for-beginners_github\demonstration\eggs\mill-scene\mill-scene.egg C:\***\3d-game-shaders-for-beginners_github\demonstration\eggs\mill-scene\mill-scene.obj
```

But end with pure model, need add textures manually, and no animations.

ref: https://www.reddit.com/r/Toontown/comments/500hek/has_anyone_converted_all_the_toontown_models_into/

Result files are in [my fork](https://github.com/gonnavis/3d-game-shaders-for-beginners/tree/three.js/demonstration/eggs/mill-scene).

[demo](http://gonnavis.com/lib/3d-game-shaders-for-beginners_github_gonnavis/demonstration/three.js/index.html)
